Scripts/travis.sh

```diff
@@ -27,7 +27,7 @@ ACTION="test"
 #  $1 : .xcodeproj file
 #  $2 : scheme to run
 #
-# The output is prettified using xcpretty and redirected to xcodebuild.log for failure analysis.
+# The output is formatted using xcpretty and redirected to xcodebuild.log for failure analysis.
 execute_xcodebuild() {
   if [ -z ${1+x} ]; then
```

Comment:
No, I don't think that's the right reason you pointed out. Here's a more detailed explanation: http://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash